device/ipv4.yaml

```diff
@@ -36,3 +34,37 @@ components:
           type: string
           format: ipv4
           default: 0.0.0.0
+          
+    # Can not derive LoopbackIpv4 from Ipv4Base as default prefix length must be 32.
+    Interface.LoopbackIpv4:
+      x-include: 
+      - '../common/common.yaml#/components/schemas/Named.Object'
+      description: >-
+        An IPv4 loopback interface
+      type: object
+      required: [interface_name]
+      properties:
+        interface_name: 
```

Comment:
Should this be ethernet_name instead?

Reply:
Yes , this is meant to be the ethernet interface behind which this loopback is connected.